lib/src/builder/generation/impl_generation.dart

```diff
@@ -727,9 +727,11 @@ class ImplGenerator {
         ..writeln('  @override')
         ..writeln('  bool get \$isClassGenerated => true;')
         ..writeln()
-        ..writeln('  /// The [ReactComponentFactory] associated with the component built by this class.')
```

Comment:
Why is this comment being removed?

Reply:
because i hate documentation. lol just kidding, i have no idea. ill fix it.